amoro-ams/pom.xml

```diff
@@ -35,6 +35,7 @@
     <url>https://amoro.apache.org</url>
 
     <properties>
+        <skip-generate-sdk>false</skip-generate-sdk>
```

Comment:
IMO, we should not generate the SDK by default as it is not required for everyone.
But the doc is necessary for the dashboard.

So we may need to separate the properties for SDK and DOC.